builder/lxd/communicator.go

```diff
@@ -92,7 +92,7 @@ func (c *Communicator) Upload(dst string, r io.Reader, fi *os.FileInfo) error {
 
 func (c *Communicator) UploadDir(dst string, src string, exclude []string) error {
 	fileDestination := fmt.Sprintf("%s/%s", c.ContainerName, dst)
-	pushCommand := fmt.Sprintf("lxc file push --debug -pr %s %s", src, fileDestination)
+	pushCommand := fmt.Sprintf("lxc file push --debug -pr %s* %s", src, fileDestination)
```

Comment:
I would maybe suggest making sure the `src` has a trailing `/` before globbing it?

Reply:
Fixed